bootloader/assets/grub.go

```diff
@@ -144,6 +151,15 @@ menuentry 'System setup' --hotkey=f 'uefi-firmware' {
 `
 
 func init() {
-	registerInternal("grub.cfg", []byte(grubBootConfig))
-	registerInternal("grub-recovery.cfg", []byte(grubRecoveryConfig))
+	// static command line for edition 1 of grub config
+	registerInternal("grub.cfg:edition=1:static_cmdline", []byte(grubBootConfigStaticCmdline))
+	registerInternal("grub.cfg",
+		[]byte(strings.Replace(grubBootConfigTemplate,
+			"####STATIC_CMDLINE####", grubBootConfigStaticCmdline, 1)))
```

Comment:
this templating is kind of dangerous,  now we have a blob that is edition one but actually is a template that we can change. It's probably time to use "go generate" or similar and have actual .cfg files around.